docs/docs/configuration-parameters.md

```diff
@@ -28,7 +28,8 @@ The configuration file should include the following fields:
   "SOARefresh": 60,
   "SOARetry":   600,
   "SOAExpire":  86400,
-  "SOAMinttl": 60
+  "SOAMinttl": 60,
+  "ContainerIPsOn": true
```

Comment:
food for thought: since container-ip's are experimental we could default this to `false` for now. user's who want to play with container-ip can set this to true if they want to play with the feature.

thoughts?